fleet_vehicle_inspection/views/fleet_vehicle_inspection_line.xml

```diff
@@ -60,6 +145,55 @@
             </form>
         </field>
     </record>
+
+    <record model="ir.ui.view" id="fleet_vehicle_inspection_line_image_form_view">
+        <field
+            name="name"
+        >fleet.vehicle.inspection.line.image.form (in fleet_vehicle_inspection)</field>
+        <field name="model">fleet.vehicle.inspection.line.image</field>
+        <field name="arch" type="xml">
+            <form string="Inspection Line Images">
+                <field name="sequence" invisible="1" />
+                <div class="row o_inspection_image_modal">
+                    <div class="col-md-6 col-xl-5">
+                        <label for="name" string="Image Name" />
+                        <h2><field name="name" placeholder="Image Name" /></h2>
+                        <label for="video_url" string="Video URL" /><br />
+                        <field name="video_url" /><br />
+                    </div>
+                    <div
+                        class="col-md-6 col-xl-7 text-center o_inspection_image_modal_container"
+                    >
+                        <div class="row">
+                            <div class="col">
+                                <field name="image_1920" widget="image" />
+                            </div>
+                            <div
+                                class="col"
+                                attrs="{'invisible': [('video_url', 'in', ['', False])]}"
+                            >
+                                <div class="o_video_container p-2">
+                                    <span>Video Preview</span>
+                                    <field
+                                        name="embed_code"
+                                        class="mt-2"
+                                        widget="video_preview"
```

Comment:
I think this widget is implemented in `website_sale` core module, can you confirm?
If so, we have a missing dependency.. but moreover I don't think we should depend on it, as it would force the installation of unwanted modules not really related to fleet.

Alternatively you could propose a `web_widget_video_preview` module in https://github.com/OCA/web/tree/14.0

Reply:
Yes .. I didn't realize that this widget had been implemented on website_sale. I also do not agree with the dependency of this module. I will test with this widget that you indicated.

Reply:
For now I will remove the inclusion of video in the inspection line and leave it on the roadmap for future implementation. But I think the idea of proposing a widget for previewing video in the oca/web repository is valid.

Reply:
:+1: